Add paid shared nodes (#333)
chainstack/docs: docs/glossary/dedicated-node.md

A dedicated node is the node you exclusively use for your production purposes.

> With a dedicated node, you pay for the compute and storage resources used by the node and do not pay for JSON-RPC requests to the node.

Reviewer: optional: Use positive language rather than negative language?
```suggestion
With a dedicated node, you pay for the compute and storage resources used by the node. The JSON-RPC requests to dedicated nodes are free of charge.
```

Author: No. It's a parallel structure and it's about explaining the difference between shared nodes and dedicated nodes as clearly as possible.

* with this node type, you pay for this and do not pay for that
* with the other node type, you pay for that and do not pay for this (a complete reverse)
